### PR TITLE
Re-enable CreateFunction test

### PR DESCRIPTION
**Pull request by abrooksv**
Re-enables the test that failed due to it was using static env var creds due to https://github.com/aws/aws-toolkit-jetbrains/pull/2193 not being done yet

Want to see if there are any other issues with these tests

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by hunterwerlla — APPROVED** [review]

**Comment by abrooksv** [comment]
Looks like we need to add `lambda:DeleteFunction`

File: jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/CliBasedStep.kt [hunk 1]
```diff
@@ -64,6 +64,7 @@ abstract class CliBasedStep : Step() {
 
     private class CliOutputEmitter(private val messageEmitter: MessageEmitter) : ProcessAdapter() {
         override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
+            LOG.debug { event.text.trim() }
```
Comment (hunterwerlla) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/CliBasedStep.kt:67]: Should we mark if it came from stdout or stderr? Also nice this was an annoying part of our tests

**Review by hunterwerlla — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/CliBasedStep.kt (hunk 1 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/CliBasedStep.kt:67]: uhhh, ya. Will add it

File: jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/upload/CreateFunctionIntegrationTest.kt [hunk 2]
```diff
@@ -76,33 +73,38 @@ class CreateFunctionIntegrationTest {
...
-        MockClientManager.useRealImplementations(disposableRule.disposable)
-
         lambdaClient = projectRule.project.awsClient()
         iamClient = projectRule.project.awsClient()
         ecrClient = projectRule.project.awsClient()
 
         lambdaName = RuleUtils.randomName()
         iamRole = iamClient.createRoleWithPolicy(RuleUtils.randomName(), DEFAULT_LAMBDA_ASSUME_ROLE_POLICY)
 
+        // Give a few secs to allow the IAM propagation
```
Comment (hunterwerlla) [inline jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/upload/CreateFunctionIntegrationTest.kt:101]: If this works can you update it to have a longer description?

**Pull request merged**